documentation/tutorials/get-started.md

```diff
@@ -116,6 +125,43 @@ end
 Note that `failure` will not necessarily ever be called. For example, `Components.Password.SignInForm` will handle
 failures internally as long as `sign_in_tokens_enabled?` is true (which is the default).
 
+#### Update formatter config
+
+Add `ash_authentication_phoenix` to .formatter.exs for auto_formatting
+**.formatter.exs**
+
+```elixir
+[
+  import_deps: [
+    :ash_authentication_phoenix, # <-------- Add this line
+    :ash_authentication,
+    :ash_postgres,
+    :ash,
+    :ecto,
+    :ecto_sql,
+    :phoenix
+  ],
+  subdirectories: ["priv/*/migrations"],
+  plugins: [Spark.Formatter, Phoenix.LiveView.HTMLFormatter],
+  inputs: ["*.{heex,ex,exs}", "{config,lib,test}/**/*.{heex,ex,exs}", "priv/*/seeds.exs"]
+]
+```
+
+#### Override mix phx.routes alias
```

Comment:
Let's not add this as phoenix 1.8 removes the need for it. Alternatively, add a note that this is only for `phoenix <= 1.7`